azure-functions/acbs-function/helpers/date.js

```diff
@@ -13,6 +13,11 @@ const addDay = (date, day) => moment(date).add({ day }).format('YYYY-MM-DD');
 const addMonth = (date, months) => moment(date).add({ months }).format('YYYY-MM-DD');
 const addYear = (date, years) => moment(date).add({ years }).format('YYYY-MM-DD');
 
+/**
+ * @returns {string} current date as ISO-8601 string (e.g. 2024-02-16T16:57:23+00:00)
+ */
+const getNowAsIsoString = () => moment().format();
```

Comment:
I'd like to replace this with
```suggestion
const getNowAsIsoString = () => formatISO(new Date());
```
but they have subtly different behaviour:

```js
(moment().format(); // 2024-02-16T16:57:23+00:00
formatISO(new Date()); // 2024-02-16T16:57:23Z

```

Reply:
Isn't +0:00, UTC and zulu time the same?